test/VariantClassifierTest.py

```diff
@@ -663,6 +664,18 @@ def test_dnp_handling(self,chr, start, end, ref, alt, vt_gt, vc_gt, sec_vc_gt, g
         self.assertTrue(codon_change == codon_change_gt, "GT/Guess: %s/%s" % (codon_change_gt, codon_change))
         self.assertTrue(protein_change == protein_change_gt, "GT/Guess: %s/%s" % (protein_change_gt, protein_change))
 
+    @TestUtils.requiresDefaultDB()
+    def test_macbeth_issue(self):
```

Comment:
Deleted.  Turned out not to be an issue.  This test was not even finished (note no assert statements).